native/ios/Integreat/AppDelegate.mm

```diff
@@ -63,6 +64,22 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 #endif
 }
 
+// for deep linking
+- (BOOL)application:(UIApplication *)application
+   openURL:(NSURL *)url
+   options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options
+{
+  return [RCTLinkingManager application:application openURL:url options:options];
+}
+
+- (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity
+ restorationHandler:(nonnull void (^)(NSArray<id<UIUserActivityRestoring>> * _Nullable))restorationHandler
+{
+ return [RCTLinkingManager application:application
+                  continueUserActivity:userActivity
+                    restorationHandler:restorationHandler];
+}
+
```

Comment:
Did deep linking not work on ios beforehand? whats this change about?

Reply:
Correct, in iOS it only worked if the app wasn't open in the background. So, two cases:
1. The app is fully closed, you tap on a deep link somewhere, the app opens and sends you to the correct link.
2. The app is open in the background, you tap on a deep link somewhere, the app opens on the page that you were last on and the deep link is otherwise ignored.

Reply:
Interesting. Thank you for the information.